model_params:
  name: 'GNM'
  model_type: "gnm"
  context_size: 5  # number of images to use as context
  len_traj_pred: 5  # number of waypoints to predict
  learn_angle: False  # bool to determine whether to learn/predict heading of the robot
  image_size: [ 85, 64 ]  # (width, height)
  normalize: True  # bool to determine whether or not normalize the waypoints
  obs_encoding_size: 1024  # size of the encoding of the observation [only used by gnm and siamese]
  goal_encoding_size: 1024  # size of the encoding of the goal [only used by gnm and siamese]
  obsgoal_encoding_size: 2048  # size of the encoding of the observation and goal [only used by stacked model]
  waypoint_spacing: 4

data_params:
  train_batch_size: 64
  train_num_workers: 16
  val_batch_size: 64
  val_num_workers: 16
  image_size: [85, 64] # width, height
  context_size: 5
  waypoint_spacing: 4
  len_traj_pred: 5

exp_params:
  LR: 0.0005
  weight_decay: 0.0
  scheduler_gamma: 0.95
  manual_seed: 1265

trainer_params:
  accelerator: 'gpu'
  devices: 1
  max_epochs: 10

logging_params:
  save_dir: "logs/"
  name: "WaypointGNM"